src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -59,6 +59,54 @@
                    Style="{StaticResource DisclaimerStyle}"
                    Visibility="{x:Bind State.Profile.IsBaseLayer}" />
 
+        <Grid Height="184">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="24" />
+                <RowDefinition Height="160" />
+            </Grid.RowDefinitions>
+            <Grid Grid.Row="0"
+                  Margin="8,0,8,0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="auto" />
+                </Grid.ColumnDefinitions>
+                <Border MinWidth="128"
+                        Background="#333333"
+                        CornerRadius="2,2,0,0">
+                    <Grid VerticalAlignment="Center">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="32" />
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="28" />
+                        </Grid.ColumnDefinitions>
+                        <IconSourceElement Grid.Column="0"
+                                           Width="16"
+                                           Height="16"
+                                           Margin="8,0,8,0"
+                                           VerticalAlignment="Bottom"
+                                           IconSource="{x:Bind State.Profile.Icon, Mode=OneWay, Converter={StaticResource IconSourceConverter}}" />
+                        <TextBlock Grid.Column="1"
+                                   VerticalAlignment="Center"
+                                   FontSize="12"
+                                   Text="{x:Bind State.Profile.TabDisplayName, Mode=OneWay}" />
+                        <IconSourceElement Grid.Column="2"
+                                           Width="16"
+                                           Height="16"
+                                           Margin="8,0,4,0"
+                                           VerticalAlignment="Bottom">
+                            <IconSourceElement.IconSource>
+                                <FontIconSource FontSize="12"
+                                                Glyph="&#xE711;" />
+                            </IconSourceElement.IconSource>
+                        </IconSourceElement>
+                    </Grid>
+                </Border>
+            </Grid>
+            <Border x:Name="ControlPreview"
+                    Grid.Row="1"
+                    Margin="8,0,8,0"
+                    HorizontalAlignment="Stretch" />

```

Comment:
dumb question: what happens if we just _always_ have this set as `HorizontalAlignment="Left"`?